demos/src/Marks/TextStyle/React/AddGlobalAttributes.ts

```diff
@@ -0,0 +1,70 @@
+import '@tiptap/extension-text-style'
+
+import { Extension, Mark } from '@tiptap/core'
+
+export const AddGlobalAttributes = Extension.create({
```

Comment:
Are those extensions required for the demo? I thought this should also work by the default Color or TextStyle extension.

Reply:
Thank you for the review. I have rewritten the demo to use the default extensions. [aaab0b2](https://github.com/ueberdosis/tiptap/pull/5897/commits/aaab0b28dd378939fdc0ebaa25e812c65ee7607f)
I mistakenly assumed it should be tested with the TextStyle extension alone.